lte/gateway/c/core/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -1030,4 +1030,35 @@ static int authenthication_t3560_handler(
   }
   OAILOG_FUNC_RETURN(LOG_NAS_AMF, RETURNok);
 }
+
+/***************************************************************************
+**                                                                        **
+** Name:    get_decrypt_imsi_suci_extension()                             **
+**                                                                        **
+** Description: Invokes .get_decrypt_imsi_info                            **
+**              to fetch decrypted imsi                                   **
+**                                                                        **
+**                                                                        **
+***************************************************************************/
+int get_decrypt_imsi_suci_extension(
```

Comment:
This is also a fine but as decryption is part of Registration can we move it there.

Reply:
ok, Done